test/Microsoft.AspNet.Mvc.Core.Test/DefaultActionSelectorTests.cs

```diff
@@ -25,8 +25,9 @@ namespace Microsoft.AspNet.Mvc
 {
     public class DefaultActionSelectorTests
     {
+        // When no actions match, MvcRouteHandler logs the message.
         [Fact]
-        public async void SelectAsync_NoMatchedActions_LogIsCorrect()
+        public async void SelectAsync_NoMatchedActions_NothingIsLogged()
```

Comment:
I wouldn't worry about writing tests that verify that nothing is logged. There are lots of code paths in MVC that don't log.